src/chat/infinite.js

```diff
@@ -314,7 +314,7 @@ export function infiniteMessageArea (dom, kb, chatChannel, options) {
         UI.media.cameraButton(dom, kb, getImageDoc, tookPicture)
       )
 
-      UI.pad.recordParticipation(chatChannel, chatChannel.doc()) // participation =
+      UI.participation.recordParticipation(chatChannel, chatChannel.doc()) // participation =
```

Comment:
I'll review this change separately, it seem unrelated to both this PR and this commit.

Reply:
Continued in https://github.com/solid/solid-ui/issues/210